testsuite/features/step_definitions/common_steps.rb

```diff
@@ -376,14 +376,16 @@
 
 Then(/^"(\d+)" channels should be enabled on "([^"]*)"$/) do |count, host|
   node = get_target(host)
-  _out, code = node.run("zypper lr -E | tail -n +5 | wc -l")
-  raise "Expected #{count} channels enabled but found #{_out}." unless count.to_i == _out.to_i
+  node.run("zypper lr -E", verbose: true)
+  out, _code = node.run("zypper lr -E | tail -n +5 | wc -l")
+  raise "Expected #{count} channels enabled but found #{out}." unless count.to_i == out.to_i
```

Comment:
I am confused, where did we get the count here? I'm aware we already had it like this, this doesn't prevent the merge, everything looks fine to me, I'm just wondering what black magic is happening

Reply:
The count?
We pass it in the scenario.

Reply:
We expect 1, but for some reason... we count 2.
I'm trying with this PR to clarify which is the second channel/repo.